test-clients/src/main/resource/contract/contracts/TokenInfoContract/TokenInfoContract.sol

```diff
@@ -35,4 +35,22 @@ contract TokenInfoContract is HederaTokenService {
 
         nonFungibleTokenInfo = retrievedTokenInfo;
     }
-}
\ No newline at end of file
+
+    function getCustomFeesForToken(address token) external returns (
+        IHederaTokenService.FixedFee[] memory fixedFees,
+        IHederaTokenService.FractionalFee[] memory fractionalFees,
+        IHederaTokenService.RoyaltyFee[] memory royaltyFees) {
+        (int responseCode,
+        IHederaTokenService.FixedFee[] memory retrievedFixedFees,
+        IHederaTokenService.FractionalFee[] memory retrievedFractionalFees,
+        IHederaTokenService.RoyaltyFee[] memory retrievedRoyaltyFees) = HederaTokenService.getTokenCustomFees(token);
+
+        if (responseCode != HederaResponseCodes.SUCCESS) {
+            revert ();
+        }
+
+        fixedFees = retrievedFixedFees;
+        fractionalFees = retrievedFractionalFees;
+        royaltyFees = retrievedRoyaltyFees;
```

Comment:
function getCustomFeesForToken(address token) external returns (
```suggestion
         IHederaTokenService.FixedFee[] memory fixedFees,
        IHederaTokenService.FractionalFee[] memory fractionalFees,
        IHederaTokenService.RoyaltyFee[] memory royaltyFees) {
        (int responseCode,
        fixedFees, fractionalFees, royaltyFees) = HederaTokenService.getTokenCustomFees(token);

        if (responseCode != HederaResponseCodes.SUCCESS) {
            revert ();
        }
    }
```

The arrays for the return results are already defined in returns(), so the assignment at the end is not needed.